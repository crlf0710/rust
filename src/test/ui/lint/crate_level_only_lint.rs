#![deny(uncommon_codepoints, unused_attributes)]

mod foo {
#![allow(uncommon_codepoints)] //~ ERROR unused lint attribute: `uncommon_codepoints`

#[allow(uncommon_codepoints)]
const BAR: f64 = 0.000001; //~^ ERROR unused lint attribute: `uncommon_codepoints`

}

#[allow(uncommon_codepoints)]
fn main() { //~^ ERROR unused lint attribute: `uncommon_codepoints`
}
