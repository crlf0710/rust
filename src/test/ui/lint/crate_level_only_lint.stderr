error: unused lint attribute: `uncommon_codepoints`
  --> $DIR/crate_level_only_lint.rs:4:10
   |
LL | #![allow(uncommon_codepoints)]
   |          ^^^^^^^^^^^^^^^^^^^
   |
note: the lint level is defined here
  --> $DIR/crate_level_only_lint.rs:1:30
   |
LL | #![deny(uncommon_codepoints, unused_attributes)]
   |                              ^^^^^^^^^^^^^^^^^
   = note: allow(uncommon_codepoints) can be specified at crate level only.

error: unused lint attribute: `uncommon_codepoints`
  --> $DIR/crate_level_only_lint.rs:6:9
   |
LL | #[allow(uncommon_codepoints)]
   |         ^^^^^^^^^^^^^^^^^^^
   |
   = note: allow(uncommon_codepoints) can be specified at crate level only.

error: unused lint attribute: `uncommon_codepoints`
  --> $DIR/crate_level_only_lint.rs:11:9
   |
LL | #[allow(uncommon_codepoints)]
   |         ^^^^^^^^^^^^^^^^^^^
   |
   = note: allow(uncommon_codepoints) can be specified at crate level only.

error: aborting due to 3 previous errors

